src/hammer-vlsi/defaults.yml

```diff
@@ -350,9 +358,22 @@ vlsi.inputs:
     #  - no_connect (Optional[bool]) - If True the bump will be present but unconnected
     #  - x (Decimal) - The X coordinate of the bump assigned to this net
     #  - y (Decimal) - The Y coordinate of the bump assigned to this net
+    #  - group (Optional[str]) - The group name for the bumps, e.g. for creating schematic symbols
     #  - custom_cell (Optional[str]) - Cell name (MACRO name) of a custom bump cell in place of the default for this bump. You must also specify the LEF and GDS for this bump in extra_libraries.
     assignments: []
 
+  # Naming scheme for the bumps' actual physical pin designators
+  # Valid options are
+  # - "A0"    (Use letters for rows, skipping I, O, Q, S, X, and Z. After Y is AA, AB, AC ...
+  #            Use numbers for columns, starting at 0, with no leading 0s (0-9, then 10, 11, ...))
+  # - "A1"    (Like A0, but numbers start at 1)
+  # - "A00"   (Like A0, but all bumps will have the same number of digits, with leading 0s appended if needed)
+  # - "A01"   (Like A00, but numbers start at 1)
+  # - "Index" (Just returns a single number in the order the bumps are listed- useful for non-grid designs)
+  # type: str
+  bumps_pin_naming_scheme: "A1"
```

Comment:
Is this arbitrary or are only those supported? e.g. could I do `A001` and have bumps like `A001`, `A002`, `B001` etc?

Reply:
No, there's no `A001`. `A01` means the digit portion will always be zero-padded such that all numbers are the same width (e.g. if there are 100 columns, the top left will be `A001`)
> all bumps will have the same number of digits

Reply:
Gotcha. That makes sense. Could we add that example into the defaults.yml documentation as well?

Reply:
done (I added this to A00 so I needed to change the example a bit to make it technically correct)